pkgs/applications/office/docear/default.nix

```diff
@@ -0,0 +1,48 @@
+{stdenv, fetchurl, runtimeShell
+, oraclejre
```

Comment:
After a *very* quick look over the website I don't see specific mention of this application requiring oracle java.

Reply:
This has been checked. One of it's components needs "sun-jre". If running with OpenJDK it fails to start.